### DISCUSSION TITLE
Skipping builds with [skip ci] or [skip cirrusci]

### DISCUSSION THREAD

**Discussion by abitrolly** [cirruslabs/cirrus-ci-docs#842] [Q&A]
I can't find if the syntax to skip builds is supported by Cirrus CI?

**Comment by RDIL** [comment]
[skip ci] and [ci skip] both work

**Reply by abitrolly** [reply]
What about `[skip cirrusci]` or `[skip cirrus]`? Any plans to add that?

**Reply by fkorotkov** [reply]
That should already work. Updated documentation in 1feaaf93a2b2dbe2c75d946c06266d15b234545f

**Reply by abitrolly** [reply]
Awesome. I will try it.

**Reply by abitrolly** [reply]
`[skip cirrus]` works. Thanks!
